app/views/users/show.html.erb

```diff
@@ -6,10 +6,10 @@
 
   <div class="profile-body profile-sections">
     <%= render "users/profile_sections/badges" %>
-    <%= render "users/profile_sections/donations_by_causes" %>
+    <!--%= render "users/profile_sections/donations_by_causes" %-->
```

Comment:
@emilyspieler why is this commented out?

Reply:
@uuunk and I ran into an issue last night with getting everything to render and were trying to figure out where the problem was. Paul and I located the issue on this line so we commented it out to see if that would fix it, which it did (somewhat) but then we decided to comment it back in after we figured out what was causing the issue. So, seeing as what got pushed is still commented out, It looks like something is going on with my commits that isn't saving my work properly. I may need to circle back to you guys once I solve that. Thanks for pointing this out, I'll investigate.